datasetinsights/estimators/faster_rcnn.py

```diff
@@ -31,6 +32,7 @@
 TRAIN = "train"
 VAL = "val"
 TEST = "test"
+NAME = "run_name"
```

Comment:
add a comment that explains what these generic constants are for, and probably should add `__` prefix since they're not meant to be used outside this module